dist/pouchdb.load.js

```diff
@@ -152,8 +152,9 @@ exports.load = utils.toPromise(function (url, opts, callback) {
       }
 
       db.info().then(function (info) {
-        var src = new db.constructor(opts.proxy,
-          utils.extend(true, {}, db.__opts, opts));
+        var srcOpts = utils.extend(true, {}, db.__opts, opts);
+        delete srcOpts.adapter;
```

Comment:
^ There are actually way more options available than `adapter`, so in my mind it would make more sense to whitelist only the keys we are interested in rather than blacklist.